packages/notifi-dapp-example/src/components/DestinationPanel.tsx

```diff
@@ -80,15 +82,12 @@ export const DestinationPanel: React.FC<DestinationPanelProps> = ({
     const senderAddress = '0xb49bbE2c31CF4a0fB74b16812b8c6B6FeEE23524';
```

Comment:
Not sure if this `senderAddress` should be the address of user's wallet address instead of hardcoded string?

Reply:
yes, actually this part is still in progress, so for this part we'll be creating a new PR once everything is sorted and tested.